ODLA/include/ODLA/odla_common.h

```diff
@@ -135,6 +135,13 @@ typedef void odla_void;
 
 //! \brief Return status
 typedef enum {
+  ODLA_DL_ERROR,
+  ODLA_FILE_NOT_EXIST,
+  ODLA_INVALID_PARAM,
+  ODLA_MEM_ERROR,
+  ODLA_NULLPTR,
```

Comment:
ODLA_NULLPTR is same as ODLA_INVALID_PARAM ?

Reply:
ok